packages/dependency-extraction-webpack-plugin/index.js

```diff
@@ -98,27 +110,31 @@ class DependencyExtractionWebpackPlugin {
 					}
 				}
 
-				// Build a stable JSON string that declares the WordPress script dependencies.
-				const sortedDepsArray = Array.from( entrypointExternalizedWpDeps ).sort();
-				const depsString = JSON.stringify( sortedDepsArray );
+				const runtimeChunk = entrypoint.getRuntimeChunk();
+
+				// Build a stable JSON string that declares the WordPress script asset.
+				const assetString = this.serialize( {
+					dependencies: Array.from( entrypointExternalizedWpDeps ).sort(),
+					version: runtimeChunk.hash,
+				} );
 
-				// Determine a filename for the `[entrypoint].deps.json` file.
+				// Determine a filename for the `[entrypoint].asset.[outputFormat]` file.
 				const [ filename, query ] = entrypointName.split( '?', 2 );
-				const depsFilename = compilation
+				const assetFilename = compilation
 					.getPath( outputFilename, {
-						chunk: entrypoint.getRuntimeChunk(),
+						chunk: runtimeChunk,
 						filename,
 						query,
 						basename: basename( filename ),
 						contentHash: createHash( 'md4' )
-							.update( depsString )
+							.update( assetString )
 							.digest( 'hex' ),
 					} )
-					.replace( /\.js$/i, '.deps.json' );
+					.replace( /\.js$/i, '.asset.' + outputFormat );
```

Comment:
This will be a breaking change for any code relying on `deps.json` currently right?

Reply:
Yes, totally. We could make it backward compatible with some flag but I think for CLI tools it's fine to introduce breaking changes.